examples/backup/outputs.tf

```diff
@@ -0,0 +1,18 @@
+##############################################################################
+# Outputs
+##############################################################################
+
+output "id" {
+  description = "RabbitMQ db instance id"
+  value       = var.rmq_db_backup_crn == null ? module.rmq_db[0].id : null
+}
+
+output "restored_rmq_db_id" {
+  description = "Restored RabbitMQ db instance id"
+  value       = module.restored_rmq_db.id
+}
+
+output "restored_rmq_db_version" {
+  description = "Restored RabbitMQ instance version"
+  value       = module.restored_rmq_db.version
+}
```

Comment:
Can we avoid the use of rqm and rqm_db and prefer the use of rabbitmq or possibly rabbitmq_database (per fscloud module name).
Maybe restored_rabbitmq_id and restored_rabbitmq_version

Reply:
Will add the example later and will use rabbitmq